x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rest/action/ReloadAnalyzerRequestBuilder.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.rest.action;
+
+import org.elasticsearch.action.support.broadcast.BroadcastOperationRequestBuilder;
+import org.elasticsearch.client.ElasticsearchClient;
+
+/**
+ * Builder for reloading of analyzers
+ */
+public class ReloadAnalyzerRequestBuilder
+        extends BroadcastOperationRequestBuilder<ReloadAnalyzersRequest, ReloadAnalyzersResponse, ReloadAnalyzerRequestBuilder> {
+
+    public ReloadAnalyzerRequestBuilder(ElasticsearchClient client, ReloadAnalyzerAction action, String... indices) {
+        super(client, action, new ReloadAnalyzersRequest(indices));
+    }
+
+}
```

Comment:
This class doesn't seem to be used, it is needed ?

Reply:
Right, it was used before with some other convenience classes for the Node client that I removed. Will also remove this one.